src/fragments/lib/storage/android/list.mdx

```diff
@@ -1,14 +1,24 @@
-You can list all of the objects uploaded under a given prefix. This will list all public files:
+You can list all of the objects uploaded under a given prefix by setting the pageSize. If the page size is set lower than the total file size available,
```

Comment:
You can list all of the objects uploaded under a given prefix by setting the pageSize. If the pagSize is set lower than the total file size available,